RecoBTag/DeepFlavour/interface/c_pf_features_converter.h

```diff
@@ -0,0 +1,263 @@
+                c_pf_features.isMu = 1;
+            }
+            // TO DO: we can do better than that by including reco::electron informations
+            c_pf_features.isEl = 0;
+            if(abs(c_pf->pdgId())==11) {
+                c_pf_features.isEl = 1;
+
+            }
+
+            c_pf_features.charge = c_pf->charge();
+            c_pf_features.lostInnerHits = deep::catch_infs(c_pf->lostInnerHits(),2);
+            c_pf_features.chi2 = deep::catch_infs_and_bound(pseudo_track.normalizedChi2(),300,-1,300);
+            //for some reason this returns the quality enum not a mask.
+            c_pf_features.quality = pseudo_track.qualityMask();
+
+            c_pf_features.drminsv = deep::catch_infs_and_bound(drminpfcandsv,0,-0.4,0,-0.4);
+
+  } 
+
+  template <typename CandidateType, typename JetType,
+            typename TrackInfoBuilderType,
+            typename ChargedCandidateFeaturesType,
+            typename VertexType>
+  void c_pf_reco_features_converter(const CandidateType * c_pf,
+                                    const JetType & jet,
+                                    const TrackInfoBuilderType & track_info,
+                                    const float & drminpfcandsv, const float & puppiw,
+                                    const int & pv_ass_quality,
+                                    const VertexType & pv, 
+                                    ChargedCandidateFeaturesType & c_pf_features) {
+
+            // track_info has to be built before passing as parameter
+
+            // could be inlined
+            float etasign = 1.;
+            if (jet.eta()<0) etasign =-1.;
+
+            c_pf_features.pt = c_pf->pt();
+            c_pf_features.eta = c_pf->eta();
+            c_pf_features.phi = c_pf->phi();
+            c_pf_features.ptrel = deep::catch_infs_and_bound(c_pf->pt()/jet.pt(),
+                                                             0,-1,0,-1);
+            c_pf_features.erel = deep::catch_infs_and_bound(c_pf->energy()/jet.energy(),
+                                                            0,-1,0,-1);
+            c_pf_features.phirel = deep::catch_infs_and_bound(fabs(reco::deltaPhi(c_pf->phi(),jet.phi())),
+                                                              0,-2,0,-0.5);
+            c_pf_features.etarel = deep::catch_infs_and_bound(fabs(c_pf->eta()-jet.eta()),
+                                                              0,-2,0,-0.5);
+            c_pf_features.deltaR = deep::catch_infs_and_bound(reco::deltaR(*c_pf,jet),
+                                                             0,-0.6,0,-0.6);
+            //c_pf_features.dxy = deep::catch_infs_and_bound(fabs(c_pf->dxy()),0,-50,50);
+
+
+            //c_pf_features.dxyerrinv = deep::catch_infs_and_bound(1/c_pf->dxyError(),
+            //                                                     0,-1, 10000.);
+
+            //c_pf_features.dxysig = deep::catch_infs_and_bound(fabs(c_pf->dxy()/c_pf->dxyError()),
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant

Reply:
not needed, they are some variables from the training framework that are not used in the model, will clean it up